sentinel-core/log/log_base.h

```diff
@@ -1,5 +1,33 @@
 #pragma once
 
+#include <string>
+
+//#include "sentinel-core/init/init_target_registry.h"
+
 namespace Sentinel {
-namespace Log {}  // namespace Log
+namespace Log {
+
+class LogBase {
+ public:
+  LogBase() = default;
+
+  static void Initialize();
+  static bool IsLogNameUsePid() { return log_name_use_pid_; }
+
+  static std::string GetLogBaseDir() { return log_base_dir_; }
+
+ private:
+  static std::string AddSeparator(std::string dir);
+
+ public:
+  static const std::string kEnvLogDir;
```

Comment:
use `static const char kEnvLogDir[];`  `std::string` is a non-POD type, static initialization will have out of order problems

https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables